packages/codemods/transforms/v3/dropdown/index.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function ({ source /*, path*/ }, { parse, visit }) {
+  const ast = parse(source);
+
+  return visit(ast, (env) => {
+    let { builders: b } = env.syntax;
+
+    return {
+      ElementNode(node) {
+        if (node.tag === 'Hds::Dropdown') {
+          // filter out the `@listPosition` attribute
+          const outputAttrs = node.attributes.filter(a => a.name !== '@listPosition');
+
+          // look up for `@listPosition`
+          const attr = node.attributes.find(a => a.name === '@listPosition')
+
+          // if `@listPosition` is `left` or `right` update the argument value
+          if (attr.value.chars === "left" || attr.value.chars === "right") {            
+            outputAttrs.push(b.attr('@listPosition', b.text(`bottom-${attr.value.chars}`)));
```

Comment:
I find this pattern ([used in Cloud UI](https://github.com/hashicorp/cloud-ui/blob/main/codemods/transforms/hds-migration/core/external-link/index.js#L84-L85)) a bit clearer/explicit:
```suggestion
            const updatedListPositionAttr = b.attr('@listPosition', b.text(`bottom-${attr.value.chars}`));
            outputAttrs.push(updatedListPositionAttr);
```